cpp/bench/ann/src/common/util.hpp

```diff
@@ -91,9 +91,15 @@ struct buf {
 
 struct cuda_timer {
  private:
+#ifndef BUILD_CPU_ONLY
   cudaStream_t stream_{nullptr};
   cudaEvent_t start_{nullptr};
   cudaEvent_t stop_{nullptr};
+#else
+  cudaStream_t stream_{0};
+  cudaEvent_t start_{0};
+  cudaEvent_t stop_{0};
+#endif
```

Comment:
```suggestion
  cudaStream_t stream_{nullptr};
  cudaEvent_t start_{nullptr};
  cudaEvent_t stop_{nullptr};
```
Not necessary anymore (merge artifact).